FBTweak/_FBEditableTweakTableViewCell.m

```diff
@@ -258,9 +261,11 @@ - (void)_updateMode:(_FBTweakTableViewCellMode)mode
     self.imageView.hidden = NO;
   } else if (_mode == _FBTweakTableViewCellModeDate) {
     _switch.hidden = YES;
-    _textField.hidden = NO;
-    _textField.enabled = NO;
+    _textField.hidden = YES;
```

Comment:
does this mean that the date is not shown?

Reply:
these are readonly! send a pic for editable tweaks

Reply:
oops here you go
![image](https://user-images.githubusercontent.com/1219975/40115505-554c2c4e-5919-11e8-84fa-7860310266a6.png)

Reply:
I think this format is nicer... I didn't touch FB implementation for the read-only though.

Reply:
I think they should be the same, don't care which format.  Note that their format shows the timezone which may be necessary.

Reply:
It used to be like this when I used [((NSDate *)value) description]...